Add `Performance Points Committee` article (#6376)
ppy/osu-wiki: wiki/Performance_points/Performance_points_commitee/en.md

---
stub: true
---

# Performance Points Committee

> The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Beatmap) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.

Reviewer: URL here seems off. Performance points have a wiki page of their own.

Author: o yeah i copied the wrong url for some reason, whoops

> The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Performance_points) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.

Reviewer 2: ```suggestion
The **Performance Points Committee** is a panel of well-versed players and developers who facilitate updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/The_Team/Developers) push changes faster and with more community coordination.
```
just playing around with the order until it felt better; "facilitated" isn't usually by itself, "made effective" is sorta vague and implies more responsibility than they have i think
and split sentence between the "what" and the "why"
and link star rating as well

Reviewer 3: "facilitate and approve updates"

> The panel was first announced to the public through the [November 2021 Performance Points & Star Rating Updates newspost](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates).

Reviewer: Should probably also mention that it was called into existence in response to the community survey from this year.

Author: added

> The panel was first announced to the public through the [November 2021 Performance Points & Star Rating Updates newspost](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to the [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) that was being held earlier that year.

Reviewer 2: ```suggestion
The panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) that was held earlier that year.
```
